src/blocks/slider/components/Slide.js

```diff
@@ -84,7 +92,7 @@ const Slide = ({
 					<Fragment>
 						<div className="wp-block-themeisle-blocks-slider-item-move-menu">
 							<Button
-								icon="arrow-left-alt2"
+								icon={ <Icon icon={chevronLeft} /> }
```

Comment:
In all icons, have space between curly braces. Not an issue but for the sake of consistency.